src/query.rs

```diff
@@ -18,7 +18,7 @@ impl RequiredSemverUpdate {
     }
 }
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, clap::ValueEnum)]
```

Comment:
This is the part I'm most unsure about. Do `--assume-semver patch` / `--assume-semver not-changed` ever make sense in the CLI? I'd assume not, so it might be better to have a separate enum at the CLI/API boundary, which is convertible into `ActualSemverUpdate` when needed.

Reply:
Why wouldn't these make sense?  Wouldn't they be useful for checking after doing a version bump?

Reply:
I also don't like that clap leaks into a non-cli-parser file, maybe we need to move this struct?

Reply:
> it might be better to have a separate enum at the CLI/API boundary, which is convertible into `ActualSemverUpdate` when needed

That was my first idea as well, but then I kind of figured that it would just be duplicate code. But if you think it would be cleaner, I'm open to change it of course :)

Reply:
Now that it is named `--release-type`, I do agree that `not-changed` makes no sense. `patch` does make sense, as `patch` and `minor` select different types of lints. `major` does indeed mot make much sense, because it is equivalent to simply not running the tool.

All of those things are only true for the current state. If #58 takes shape, all of those will make sense.

I'd still remove the `not-changed` because `--release-type not-changed` simply doesn't make sense to me, from how it is worded.

Reply:
Fixed